app/components/explore/side-bar.js

```diff
@@ -1,155 +1,175 @@
 import Component from '@ember/component';
 import moment from 'moment';
-import { computed } from '@ember/object';
+import { computed, action, get } from '@ember/object';
```

Comment:
@Anupam-dagar `get` is used as -
```
import { get } from '@ember/object';
get(obj, "name");
```
Please refer to https://api.emberjs.com/ember/release/functions/@ember%2Fobject/get for.mpre understanding. I don't think you need this here

Reply:
I confused this get with the one used in computed property. Thanks